AdvancedPlaybackSampleApp/build.gradle

```diff
@@ -9,7 +9,7 @@ buildscript {
         javaVersion = JavaVersion.VERSION_1_8
 
         // App dependencies
-        exoplayerVersion = '2.9.5'
+        exoplayerVersion = '2.9.3'
```

Comment:
I'm not sure about this line. Could you say, what is the aim to keep exoplayerVersion on sample level, when it is located on project level? I suppose on of them is redundant.